### PR TITLE
Remove actions to install MPI, Singularity and Nvidia GPU drivers

### PR DESCRIPTION
**Pull request by jaimesouza**
## Description

 It aims to clean up slurmd, removing unnecessary actions (e.g. MPI, Singularity, and Nvidia GPU drivers)

## How was the code tested?

The was code was tested using the GitHub actions test workflow.

## Related issues and/or tasks

ASP-2709

## Checklist

- [x] I am the author of these changes, or I have the rights to submit them.
- [ ] I have added the relevant changes to the README and/or documentation.
- [x] I have self reviewed my own code.
- [x] All requested changes and/or review comments have been resolved.

### CODE REVIEW

**Review by jamesbeedy — APPROVED** [review]
+1

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ops==1.3.0
 etcd3gw==1.0.2
-git+https://github.com/omnivector-solutions/slurm-ops-manager.git@0.8.7
\ No newline at end of file
+git+https://github.com/omnivector-solutions/slurm-ops-manager.git@0.8.9
```
Comment (NucciTheBoss) [inline requirements.txt]: Just a clerical question. What changed between slurm_ops_manager 0.8.7 to slurm_ops_manager 0.8.9

**Review by NucciTheBoss — APPROVED** [review]
Looks good to me. Thanks for handling this @jaimesouza!

File: requirements.txt (hunk 1 above)
Comment (jaimesouza) [reply requirements.txt]: Hi @NucciTheBoss! It basically drops support for MPI, Singularity and Nvidia installations (https://github.com/omnivector-solutions/slurm-ops-manager/pull/88) and correct default path for plugstack conf (https://github.com/omnivector-solutions/slurm-ops-manager/pull/86)

slurm-ops-manager 0.8.10 applies a small fix (https://github.com/omnivector-solutions/slurm-ops-manager/pull/89)

**Comment by NucciTheBoss** [comment]
Sounds good to me. Thank you for the explanation!

**Pull request merged**